src/langcheck/augment/en/_gender/_gender.py

```diff
@@ -0,0 +1,122 @@
+from __future__ import annotations
+
+from collections.abc import Iterable
+from random import choice
+from types import MappingProxyType
+
+import nltk
+from nltk import pos_tag, word_tokenize
+from nltk.tokenize.treebank import TreebankWordDetokenizer
+
+from langcheck.augment.en._gender._gender_pronouns import (_PRONOUNS_DICT,
+                                                           _BaseGenderPronouns)
+
+try:
+    nltk.data.find('averaged_perceptron_tagger')
+except LookupError:
+    nltk.download('averaged_perceptron_tagger')
```

Comment:
When I ran the code in a brand new dev container, I got the following error. Do we need to add `nltk.download('punkt')` too?

```
>>> langcheck.augment.en.gender(sentence)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/workspaces/langcheck/src/langcheck/augment/en/_gender/_gender.py", line 116, in gender
    return [_replace_gender_pronouns(texts, target_gender)]
  File "/workspaces/langcheck/src/langcheck/augment/en/_gender/_gender.py", line 79, in _replace_gender_pronouns
    tagged_words = pos_tag(word_tokenize(text))
  File "/home/vscode/.local/lib/python3.8/site-packages/nltk/tokenize/__init__.py", line 129, in word_tokenize
    sentences = [text] if preserve_line else sent_tokenize(text, language)
  File "/home/vscode/.local/lib/python3.8/site-packages/nltk/tokenize/__init__.py", line 106, in sent_tokenize
    tokenizer = load(f"tokenizers/punkt/{language}.pickle")
  File "/home/vscode/.local/lib/python3.8/site-packages/nltk/data.py", line 750, in load
    opened_resource = _open(resource_url)
  File "/home/vscode/.local/lib/python3.8/site-packages/nltk/data.py", line 876, in _open
    return find(path_, path + [""]).open()
  File "/home/vscode/.local/lib/python3.8/site-packages/nltk/data.py", line 583, in find
    raise LookupError(resource_not_found)
LookupError: 
**********************************************************************
  Resource punkt not found.
  Please use the NLTK Downloader to obtain the resource:

  >>> import nltk
  >>> nltk.download('punkt')
  
  For more information see: https://www.nltk.org/data.html

  Attempted to load tokenizers/punkt/PY3/english.pickle

  Searched in:
    - '/home/vscode/nltk_data'
    - '/usr/local/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/local/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
    - ''
**********************************************************************
```

Reply:
I added the download step. PTAL.